crafting-dead-immerse/src/main/java/com/craftingdead/immerse/world/action/BuildAction.java

```diff
@@ -133,7 +134,7 @@ public void stop(StopReason reason) {
     if (!reason.isCompleted()) {
       return;
     }
-
+    //TODO Maybe send block place events on the forge event bus? - juan
```

Comment:
No not really needed, the event is intended more for placing an individual block with an item.